packages/@expo/cli/src/prebuild/updateFromTemplate.ts

```diff
@@ -95,8 +95,16 @@ async function cloneTemplateAndCopyToProjectAsync({
   exp: Pick<ExpoConfig, 'name' | 'sdkVersion'>;
   platforms: ModPlatform[];
 }): Promise<string[]> {
+  const platformDirectories = unknownPlatforms
+    .map((platform) => {
+      return `./${platform}`;
+    })
```

Comment:
Small suggestion, again, not blocking merging:

```suggestion
    .map((platform) => `./${platform}`)
```